libdrgn/type.h

```diff
@@ -348,8 +364,9 @@ drgn_compound_type_builder_add_member(struct drgn_compound_type_builder *builder
  * member must remain valid for the lifetime of @c builder->prog.
  * @param[in] tag Name of the type. Not copied; must remain valid for the
  * lifetime of @c builder->prog. May be @c NULL if the type is anonymous.
- * @param[in] size Size of the type in bytes.
+ * @param[in] size Size of the type in bytes. Ignored if type is incomplete.
```

Comment:
Now that I think about it, I think we want to explicitly return an error if `is_complete` and `size != 0`. We should also return an error if `is_complete` and members were added to the builder.